src/agent/rustjail/src/seccomp.rs

```diff
@@ -39,6 +39,20 @@ fn get_rule_conditions(args: &[LinuxSeccompArg]) -> Result<Vec<ScmpArgCompare>>
     Ok(conditions)
 }
 
+pub fn check_unknown_syscalls(scmp: &LinuxSeccomp) -> String {
+    let mut unknown_syscalls = String::new();
```

Comment:
This might be better as a `Vec<String>` (or `Vec<&str>`).

Reply:
I fixed it. Thank you.